GVFS/GVFS.Common/NuGetUpgrade/InstallActionInfo.cs

```diff
@@ -36,5 +38,10 @@ public InstallActionInfo(string name, string version, string args, string instal
         /// The version of the component that this entry installs
         /// </summary>
         public string Version { get; }
+
+        /// <summary>
+        /// The command to run. If this is present, the installer relative path is not used.
```

Comment:
> "the installer relative path is not used"

Is this because it's not needed or something else is used instead? (I'm not really sure what that part of the comment means in general)